heltour/tournament/admin.py

```diff
@@ -544,7 +543,7 @@ def review_nominated_games_pgn_view(self, request, object_id):
         pgn = lichessapi.get_pgn_with_cache(gameid, priority=10)
 
         # Strip most tags for "blind" review
-        pgn = re.sub('\[[^R]\w+ ".*"\]\n', '', pgn)
+        pgn = re.sub('\\[[^R]\\w+ ".*"\\]\n', '', pgn)
```

Comment:
but I think the pythonic way of doing this is using a regrex string literal:
```suggestion
        pgn = re.sub(r'\[[^R]\w+ ".*"\]\n', '', pgn)
```
> The solution is to use Python’s raw string notation for regular expression patterns; backslashes are not handled in any special way in a string literal prefixed with 'r'. So r"\n" is a two-character string containing '\' and 'n', while "\n" is a one-character string containing a newline. Usually patterns will be expressed in Python code using this raw string notation.

https://docs.python.org/3/library/re.html